### PR TITLE
TASK: Adjust Neos to work with PHP 7.2

### PR DESCRIPTION
**Pull request by kdambekalns**
This enables PHP 7.2 testing on Travis CI, fixes a functional test that broke
and adjusts one actual code error.

### CODE REVIEW

**Comment by kdambekalns** [comment]
Uah, why is Travis pulling an old Flow? I guess I'll restart the job after some time…

File: Neos.ContentRepository/Classes/TypeConverter/NodeTemplateConverter.php [hunk 1]
```diff
@@ -77,7 +77,7 @@ class NodeTemplateConverter extends NodeConverter
      * @return mixed An object or \Neos\Error\Messages\Error if the input format is not supported or could not be converted for other reasons
      * @throws \Exception
      */
-    public function convertFrom($source, $targetType, array $subProperties = array(), PropertyMappingConfigurationInterface $configuration = null)
+    public function convertFrom($source, $targetType = null, array $subProperties = array(), PropertyMappingConfigurationInterface $configuration = null)
```
Comment (bwaidelich) [inline Neos.ContentRepository/Classes/TypeConverter/NodeTemplateConverter.php]: Huh, but that isn't in sync with the interface's signature!? https://github.com/neos/flow-development-collection/blob/master/Neos.Flow/Classes/Property/TypeConverterInterface.php#L113

Comment (dfeyer) [reply Neos.ContentRepository/Classes/TypeConverter/NodeTemplateConverter.php]: It match the parent class ... but not the interface, look like a PHP bug ;)

Comment (bwaidelich) [reply Neos.ContentRepository/Classes/TypeConverter/NodeTemplateConverter.php]: Ah, I see.. But that makes sense I guess: A subclass can make the argument optional but not the other way around

Comment (kitsunet) [reply Neos.ContentRepository/Classes/TypeConverter/NodeTemplateConverter.php]: Yes, sounds ok

Comment (kdambekalns) [reply Neos.ContentRepository/Classes/TypeConverter/NodeTemplateConverter.php]: https://platform.sh/blog/watch-your-inheritance

**Pull request merged**